frontend/src/app/main/ui/workspace/tokens/context_menu.cljs

```diff
@@ -372,8 +379,11 @@
 (mf/defc menu-tree
   [{:keys [selected-shapes submenu-offset type errors] :as context-data}]
   (let [shape-types (into #{} (map :type selected-shapes))
+        editing-ref (mf/deref refs/workspace-editor-state)
+        editing?    (some? editing-ref)
```

Comment:
Bug: once you open the text editor once, this does not reset: even if you close the editor, you cannot apply tokens from now on, until you exit the file or reload the page.